README.md

```diff
@@ -0,0 +1,67 @@
+
+
+# Anmies Facts App
+
+> "Anmies Facts" is a SPA to show Anmies and retrieve information about each one of them.
+
+- Display a list of 13 Animes.
+- Click in one to see more information.
+
+![image](https://user-images.githubusercontent.com/43178495/143657151-b825bc54-2fc9-4c96-aeff-704b59dc2b62.png)
+![image](https://user-images.githubusercontent.com/43178495/143657248-7b75a5b4-434d-4237-90de-f3507dabd027.png)
+
+## Built With
+
+- React
+- Redux
+- Rect-Bootstrap
+- React-Icons
+- CSS3
+- Netlify
+- [anime-facts-rest-api](https://chandan-02.github.io/anime-facts-rest-api/)
+
+## Live Demo
+[Live Demo](https://animes-facts.netlify.app/)
+
+## Video link
+[Video](https://www.loom.com/share/a06925cfee6b4ea99dfafad02f70fb09)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup and Install
+
+- Clone this repository using the link above (click on the 'code' button)
+- Open a terminal and `cd` to the cloned repository
+- run `npm install`
+- run `npm run start` to open a live server in your preferred browser.
+ 
```

Comment:
- [ ] _[Optional]_ It would be nice if you add a section for how to start testing your app, for your nontechnical readers :nerd_face: